### PR TITLE
Update CFB4 for Grand Finals

### PR DESCRIPTION
**Pull request by fajar13k**
- Add Grand Finals match schedules
- Add Grand Finals mappool
- Add Finals match results

### CODE REVIEW

File: wiki/Tournaments/CFB/4/en.md [hunk 1]
```diff
@@ -77,22 +77,48 @@ The Catch French Beginner 4 is run by various community members.
...
+## Match schedule: Grand Finals
 
-Saturday, 23 May 2020:
+Saturday, 30 May 2020:
 
 | Match ID | Player A | Player B | Match time (UTC+2) |
 | :-: | --: | :-- | :-: |
-| #31 | Remyria ![][flag_CA] | ![][flag_FR] Hecatia15 | **19:00** |
+| #38 | Remyria ![][flag_CA] | ![][flag_FR] Hecatia15 | **19:00** |
+| #39¹ | Hecatia ![][flag_FR] | ![][flag_CA] Remyria | **20:00** |
```
Comment (Joehuu) [inline wiki/Tournaments/CFB/4/en.md:87]:
```suggestion
| #39¹ | Hecatia15 ![][flag_FR] | ![][flag_CA] Remyria | **20:00** |
```

File: wiki/Tournaments/CFB/4/fr.md [hunk 2]
```diff
@@ -77,25 +77,51 @@ La Catch French Beginner 4 ést gérée par différents membres de la communaut
...
+## Planning des matches: Grandes Finales
 
-Samedi, 23 Mai 2020:
+Samedi, 30 Mai 2020:
 
 | ID du match | Joueur 1 | Joueur 2 | Horaire du match (UTC+2) |
 | :-: | --: | :-- | :-: |
-| #31 | Remyria ![][flag_CA] | ![][flag_FR] Hecatia15 | **19:00** |
+| #38 | Remyria ![][flag_CA] | ![][flag_FR] Hecatia15 | **19:00** |
+| #39¹ | Hecatia ![][flag_FR] | ![][flag_CA] Remyria | **20:00** |
```
Comment (Joehuu) [inline wiki/Tournaments/CFB/4/fr.md:87]:
```suggestion
| #39¹ | Hecatia15 ![][flag_FR] | ![][flag_CA] Remyria | **20:00** |
```

File: wiki/Tournaments/CFB/4/en.md [hunk 3]
```diff
@@ -210,6 +236,21 @@ Sunday, 24 May 2020:
...
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Remyria** ![][flag_CA] | **7** | 4 | ![][flag_FR] Hecatia15 | [#1](https://osu.ppy.sh/community/matches/62013551) |
+
+Sunday, 24 May 2020:
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Trifano** ![][flag_FR] | **7** | 2 | ![][flag_FR] Madeline  | [#1](https://osu.ppy.sh/community/matches/62051294) |
```
Comment (Joehuu) [inline wiki/Tournaments/CFB/4/en.md:251]:
```suggestion
| **Trifano** ![][flag_FR] | **7** | 2 | ![][flag_FR] Madeline | [#1](https://osu.ppy.sh/community/matches/62051294) |
```

File: wiki/Tournaments/CFB/4/fr.md [hunk 4]
```diff
@@ -210,6 +236,21 @@ Dimanche, 24 Mai 2020:
...
+
+| Joueur 1 |  |  | Joueur 2 | Lien de match |
+| --: | :-: | :-: | :-- | :-- |
+| **Remyria** ![][flag_CA] | **7** | 4 | ![][flag_FR] Hecatia15 | [#1](https://osu.ppy.sh/community/matches/62013551) |
+
+Dimanche, 24 Mai 2020:
+
+| Joueur 1 |  |  | Joueur 2 | Lien de match |
+| --: | :-: | :-: | :-- | :-- |
+| **Trifano** ![][flag_FR] | **7** | 2 | ![][flag_FR] Madeline  | [#1](https://osu.ppy.sh/community/matches/62051294) |
```
Comment (Joehuu) [inline wiki/Tournaments/CFB/4/fr.md:251]:
```suggestion
| **Trifano** ![][flag_FR] | **7** | 2 | ![][flag_FR] Madeline | [#1](https://osu.ppy.sh/community/matches/62051294) |
```

**Comment by cl8n** [comment]
@fajar13k if the beatmap links come from a program you should update it to fix fe78f006d8b15bedbc7c25b28ad130dc9bd7e017

**Pull request merged**